flat_mutation_reader.hh

```diff
@@ -363,19 +368,26 @@ public:
     class filter {
     private:
         std::function<bool (const dht::decorated_key&)> partition_filter;
+        std::function<bool (const mutation_fragment&)> mutation_filter;
     public:
-        filter(std::function<bool (const dht::decorated_key&)>&& pf = [] (const dht::decorated_key&) { return true; })
+        filter(std::function<bool (const dht::decorated_key&)>&& pf = [] (const dht::decorated_key&) { return true; },
+               std::function<bool (const mutation_fragment&)>&& mf = [] (const mutation_fragment&) { return true; })
             : partition_filter(std::move(pf))
+            , mutation_filter(std::move(mf))
```

Comment:
Again, this term tells me this is filtering partitions also. `mutation` == `partition`.

Reply:
Will rename to `mutation_fragment_filter`